kong/db/declarative/init.lua

```diff
@@ -802,6 +802,38 @@ function declarative.load_into_cache_with_events(entities, meta, hash)
     return nil, "failed to persist cache page number inside shdict: " .. err
   end
 
+  local worker_count = ngx.worker.count()
+  local sleep_left = 60
+  local sleep_time = 0.0375
```

Comment:
Might be a dumb question... But where does the magic number come from?

Reply:
Sleeps are:
75 ms
150 ms
300 ms
600 ms
1200 ms
2400 ms
4800 ms
...

worker events are polled by default once per second. On my testing it was quite hood start number. It comes from slight testing but in reality it could be anything else.